pkgs/tools/misc/yt-dlp/default.nix

```diff
@@ -0,0 +1,69 @@
+{ lib, fetchurl, buildPythonPackage
+, zip, ffmpeg, rtmpdump, phantomjs2, atomicparsley, pycryptodome, pandoc
+, fetchFromGitHub
+, websockets, mutagen
+# Pandoc is required to build the package's man page. Release tarballs contain a
+# formatted man page already, though, it will still be installed. We keep the
+# manpage argument in place in case someone wants to use this derivation to
+# build a Git version of the tool that doesn't have the formatted man page
+# included.
```

Comment:
Can we always generate it to make sure it is up to date?

Reply:
Actually we're not fetching release tarballs since the package fetches directly from github, so I should remove that comment